tests/testapp/settings.py

```diff
@@ -160,3 +160,4 @@
 }
 
 TEST_RUNNER = 'django_nose.NoseTestSuiteRunner'
+USE_TZ = False
```

Comment:
This silenced a bunch of warnings in executing the test suite. It would be good to test `__lt`, `__gt`, etc. for datetimes with time zones at some point.